src/demos/Welcome/main.vala

```diff
@@ -0,0 +1,11 @@
+Gtk.Builder builder;
+
+public void set_builder (Gtk.Builder b) {
+	builder = b;
+}
```

Comment:
I'm ok with using `builder` instead of `workbench.builder` but is it possible to make it so that this isn't needed?

it's going to be confusing to beginners 

---

EDIT

We might also want to expose the preview window so actually

`workbench.builder` and `workbench.window` would be nice IMHO

Reply:
I am planning to put that confusing boilerplate into a separate file, that gets compiled together, so you still can easily use them.